kong/plugins/key-auth/handler.lua

```diff
@@ -25,6 +25,11 @@ local function load_credential(key)
     return nil, err
   end
 
+  if cred.ttl == 0 then
```

Comment:
The description talks between 0 and 1, should this then be:

```suggestion
  if cred.ttl < 1 then
```

Reply:
`cred.ttl` is the result of `floor`. So theorily, it should be `cred.ttl == 0`.